app/src/main/res/values/donottranslate.xml

```diff
@@ -3,5 +3,5 @@
   -->
 
 <resources>
-  <string name="plaid_public_key">TODO ADD YOUR KEY HERE</string>
+  <string name="plaid_link_token">TODO ADD YOUR LINK TOKEN HERE</string>
```

Comment:
Actually, we should remove this file entirely. `strings.xml` is intended for static strings. Things like the token are generated each time so they wouldn't be appropriate here.

Reply:
⚡  Gotcha, makes sense!